Source/WebCore/platform/graphics/TrackPrivateBase.h

```diff
@@ -50,7 +53,8 @@ class WEBCORE_EXPORT TrackPrivateBase
 public:
     virtual ~TrackPrivateBase() = default;
 
-    virtual TrackPrivateBaseClient* client() const = 0;
+    size_t addClient(TrackPrivateBaseClient::Dispatcher&&, TrackPrivateBaseClient&);
+    void removeClient(uint32_t); // Can be called multiple times with the same id.
```

Comment:
Would really be nice to use ObjectIdentifier for identifiers. Also, earlier the id was a size_t and here it's an uint32_t? Would be nice for the PR to be consistent.